src/api/topic.ts

```diff
@@ -0,0 +1,41 @@
+import { request, toResponse } from '../request'
+import type { Topic } from '../types/api-responses'
+import type { PaginationOption } from '../types/options'
+
+/**
+ * 获取圈子动态
+ * @param topicId 圈子 id
+ * @param option 选项
+ */
+export const getTabsSquareFeed = <T = Topic.GetTabsSquareFeedResponse>(
+  topicId: string,
+  option: PaginationOption = {}
```

Comment:
Please add a type for `PaginationOption`

Reply:
done